app/models/notification.rb

```diff
@@ -1,11 +1,11 @@
 class Notification < ApplicationRecord
-
-  default_scope->{order(created_at: :desc)} # 新着順
-
   belongs_to :visiter, class_name: "Member", foreign_key: "visiter_id", optional: true
   belongs_to :visited, class_name: "Member", foreign_key: "visited_id", optional: true
 
   belongs_to :room, optional: true
   belongs_to :message, optional: true
 
+  def self.update_checked!
+    where(checked: false).each {|notification| notification.update_attributes(checked: true)}
```

Comment:
このメソッドもメソッド名と実際の処理の挙動が異なってしまっていますね
`update` と `update!` は挙動が違います！

Reply:
よくよく考えたら下記のコードで良いように思えてきました

```suggestion
    where(checked: false).update_all(checked: true)
```

Reply:
https://api.rubyonrails.org/classes/ActiveRecord/Relation.html#method-i-update_all